comms/discovery/misc/hashids_util.go

```diff
@@ -20,6 +20,15 @@ func DecodeHashId(id string) (int, error) {
 	}
 	return ids[0], err
 }
+
 func EncodeHashId(id int) (string, error) {
 	return hashIdUtil.Encode([]int{id})
 }
+
+func MustEncodeHashID(id int) string {
+	enc, err := EncodeHashId(id)
+	if err != nil {
+		panic(err)
```

Comment:
Will this bring down the server?

Reply:
Must prefix implies it will panic... so it's just used in tests for convenience